sycl/tools/sycl-xocc/bin/sycl-xocc

```diff
@@ -37,6 +37,9 @@
 # the environment variables are a quick workaround to swap boards.
 usage() { echo xocc_compile: error: $2 >&2; exit $1; }
 
+# Make this a parameter passed in from Clang at some point
+export VERBOSE=1
```

Comment:
Perhaps it is time to document the options/environment variables of the tool somewhere?

Reply:
I think a lot of it should be pushed down through the compiler: https://github.com/triSYCL/sycl/issues/25 rather than be environment variables. I think VERBOSE is the only environment variable we don't share with XRT/SDAccel e.g. XILINX_XRT and XCL_EMULATION_MODE. 
The other environment variables in that file shouldn't be altered by

Reply:
I am of course not against adding a .MD explaining the script in another patch though as that's helpful for everyone